lime/root/views/fancyform.html

```diff
@@ -11,6 +11,7 @@
 <script type='text/javascript'>
   $(document).ready(function() {
     LIME.flash();
+    $('#login #anchor').val((location.href.split("#")[1] || ""));
```

Comment:
I think we should probably extract this from the URL in the endpoint and put it the form via in the template.